### PR TITLE
My version of palindrome from intro module

### PR DESCRIPTION
**Pull request by WBigaj**
So I managed to create pull request with palindrome on new branch with only palindrome. Thanks for DamiXioR,danielzwierz, Vicx95 and m-kop for feedback for both palindrome and homework from module2.

### CODE REVIEW

File: Palindrome/Palindrome.cpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    return result;
+}
+
+int main() {
+    std::cout << isPalindrome("Ala ma kota") << "\n";
+    std::cout << isPalindrome("ala") << "\n"; 
+    std::cout << isPalindrome("kajak") << "\n";
+    std::cout << isPalindrome("neveroddoreven") << "\n";
+    std::cout << isPalindrome("abacada") << "\n";
+}
```
Comment (kyrtaq) [inline Palindrome/Palindrome.cpp:20]: always add a new line

File: Palindrome/Palindrome.cpp [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+
+bool isPalindrome (const std::string& text) {
+    bool result = true;
+    for (long unsigned int i = 0;i< text.length() / 2;i++) {
```
Comment (kyrtaq) [inline Palindrome/Palindrome.cpp:5]: better to use size_t instead of long unsigned int

File: Palindrome/Palindrome.cpp [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+
```
Comment (kyrtaq) [inline Palindrome/Palindrome.cpp:2]: if you want to use strings you should #include <string>

File: Palindrome/Palindrome.cpp [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    }
+    return result;
+}
+
+int main() {
+    std::cout << isPalindrome("Ala ma kota") << "\n";
+    std::cout << isPalindrome("ala") << "\n"; 
+    std::cout << isPalindrome("kajak") << "\n";
+    std::cout << isPalindrome("neveroddoreven") << "\n";
+    std::cout << isPalindrome("abacada") << "\n";
```
Comment (kyrtaq) [inline Palindrome/Palindrome.cpp:19]: you dont have to use "\n" you can use it as a one character '\n' it works the same but doesnt generate '\0'

File: Palindrome/Palindrome.cpp (hunk 1 above)
Comment (kyrtaq) [inline Palindrome/Palindrome.cpp:20]: you specified that main returns int, but you didnt return it, i would suggest to put return 0; or change it to void main

Comment (piezol) [reply Palindrome/Palindrome.cpp:20]: @kyrtaq void main is C, not C++ :)

**Comment by ziobron** [comment]
Thanks @WBigaj. There were no additional points for this task, but thank you for your implementation. Let me close it now :)

**Pull request closed without merging**